src/main/java/modtrekt/logic/commands/AddTaskCommand.java

```diff
@@ -73,6 +73,11 @@ public CommandResult execute(Model model) throws CommandException {
         if (modCode != null && !model.hasModuleWithModCode(modCode)) {
             throw new CommandException(String.format("Module code %s does not exist", modCode));
         }
+        // If module is marked as done, user can't add tasks
+        if (modCode != null && !model.hasModuleWithModCode(modCode)
+                && model.parseModuleFromCode(modCode).isDone()) {
+            throw new CommandException(String.format("Module code %s is marked as done!", modCode));
+        }
```

Comment:
I don't think this change is allowed by the feature freeze because it changes program behavior - it's not a bug, just a feature flaw.